.github/CONVENTIONS.md

```diff
@@ -0,0 +1,132 @@
+
+### Shared directories
+For some asset types there might be many of the same type. E.g. mutliple kinds of food like `Apple`, `Orange`, `Plum`. They are allowed to be put into one directory, then also sharing the same `Data` folder, as they are distinct by their names. If there is a logical grouping of many sub variants that belong together they can of course be put into a seperate sub folder that has its own `Data` folder then.
+
+### Data directory
+On import the game automatically creates a data folder for textures and materials used. Due to a [bug](https://feedback.bistudio.com/T165764) in the workbench the casing is incorrect. Always **manually** create a `Data` folder in the directory you want to import the object into.
+
+### File types
+Only use `.fbx` and `.tiff`. Those source files must be included alongside the imported asset and follow their naming.  
+E.g. `ExampleObject_01.xob` + `ExampleObject_01.fbx` + `ExampleObject_01_BCR.tiff`.
+
+### File naming
+The naming scheme of assets can be derived from [Editor Entity Naming Conventions](https://community.bistudio.com/wiki/Arma_Reforger:Editor_Entity_Naming_Conventions) ignoring part 1, 2 and 7 on the first table. 
+An example could be:
+- `MyObject_01.xob` (Mesh object)
+- `MyObject_Base.et` (Add a base class to prefabs if it is likely that there will be many variants with a shared setup)
+- `MyObject_01_Large_Blue.et` (Inherits from `MyObject_Base.et` is scaled to be big and has a blue material assigned.)
+
+### File suffixes
+Textures and materials need to be suffixed according to the slot they are made for e.g. `Name_BCR.edds`, `Name_NMO.edds`. The suffix serves a technical purpose for the game to select compression algorithms.  
+Check out the vanilla game assets for reference and also read the [guide on texture creation](https://community.bistudio.com/wiki/Arma_Reforger:Textures). 
+The [Arma Reforger Discord](https://discord.gg/arma) contains additional (often pinned) information. For textures (and materials) check out `#enfusion_texture_markers`
+- [Texture compression explaination](https://discordapp.com/channels/105462288051380224/976231628785475624/981871081046569000)
+- [Devs use TIFF with 8Bits and LZW](https://discordapp.com/channels/105462288051380224/976231628785475624/977200057357074432)
+- [Substance painter export remarks](https://discordapp.com/channels/105462288051380224/976231628785475624/977154368526753812)
+
+### File sizes
+One important factor in deciding if an asset will be included is its filesize. Because of the size limitations for console mods the framework needs to shave off every KB it can to give the individual servers as much room for addtional asset mods as possible. Metrics for triangle counts and texture sizes depend heavily on the pyhsical size and if it is a hand helt item or not.
```

Comment:
```suggestion
One important factor in deciding if an asset will be included is its file size. Because of the size limitations for console mods the framework needs to shave off every KB it can to give the individual servers as much room for additional asset mods as possible. Metrics for triangle counts and texture sizes depend heavily on the physical size and if it is a hand held item or not.
```